### PR TITLE
gnome2.gnome_vfs: remove

### PR DESCRIPTION
**Pull request by jtojnar**
## Description of changes

Depends on: #322832, #322599, #322831, https://github.com/NixOS/nixpkgs/pull/324173
Part of #39976

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
  - [x] verbiste-gtk
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -29939,10 +29939,6 @@ with pkgs;
 
   coreth = callPackage ../applications/networking/coreth { };
 
-  coriander = callPackage ../applications/video/coriander {
```
Comment (SuperSandro2000) [inline pkgs/top-level/all-packages.nix:29942]: Please add an alias entry for this

File: pkgs/applications/misc/verbiste/default.nix [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
 stdenv.mkDerivation rec {
   pname = "verbiste";
 
-  version = "0.1.47";
+  version = "0.1.48";
 
   src = fetchurl {
     url = "https://perso.b2b2c.ca/~sarrazip/dev/${pname}-${version}.tar.gz";
-    sha256 = "02kzin3pky2q2jnihrch8y0hy043kqqmzxq8j741x80kl0j1qxkm";
+    sha256 = "qp0OFpH4DInWjzraDI6+CeKh85JkbwVYHlJruIrGnBM=";
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/verbiste/default.nix:10]:
```suggestion
    hash = "sha256-qp0OFpH4DInWjzraDI6+CeKh85JkbwVYHlJruIrGnBM=";
```

File: pkgs/top-level/all-packages.nix [hunk 3]
```diff
@@ -36574,10 +36574,6 @@ with pkgs;
 
   graphwar = callPackage ../games/graphwar { };
 
-  gtetrinet = callPackage ../games/gtetrinet {
```
Comment (SuperSandro2000) [inline pkgs/top-level/all-packages.nix]: here we need an alias entry, too

**Comment by jtojnar** [comment]
Resolved and extracted the leaves to https://github.com/NixOS/nixpkgs/pull/322986

**Comment by SuperSandro2000** [comment]
I think we can merge this when targeting staging.

**Comment by jtojnar** [comment]
Ugh, looks like another dependent was introduced in the meanwhile: https://github.com/NixOS/nixpkgs/pull/278361

Opened https://github.com/NixOS/nixpkgs/pull/324173

**Review by SuperSandro2000 — APPROVED** [review]

**Pull request merged**